docs/content/get-started/build-workers.md

```diff
@@ -13,9 +13,16 @@ As you will be launching a cloud deployment, you need to build out the code exec
   <br/>**NOTE:** Building workers for the first time may take a while (about 10 minutes). Why not make yourself a cup of tea or check out [our Youtube channel](https://www.youtube.com/channel/UC7BE8B2yUeQxPvZytk47NYw/videos) while you wait?
   <br/>
   
-3\.  **Your workers have finished building when:** You see the following message in the Unity Editor's Console window: `Completed build for Cloud target`. 
+> When you build your workers you may see the following errors in the Unity console: 
+> <br/>`The worker "iOSClient" cannot be built for a Cloud deployment: your Unity Editor is missing build support for iOS. Please add the missing build support options to your Unity Editor.`
+> <br/><br/>`The worker "AndroidClient" cannot be built for a Cloud deployment: your Unity Editor is missing build support for Android.Please add the missing build support options to your Unity Editor.`<br/>
+> <br/>These errors can be ignored if you are not developing a game for Android or iOS
```

Comment:
I've actually edited this on Jamie's branch to make it active voice and imperative so it's consistent with the rest of the text, as well a moved it where the build errors are below. (I think it makes sense as part of the build errors information?!)

This is in a set of instructions for the getting started using the GDK; our user is just following instructions; they are not developing a game for Android, so the text doesn't really make sense as-is.
I think the text on Jamie's branch (wh

Reply:
If I merge prior to Jamie that should land regardless of my PR. I agree that this could be more active and succinct.